ezgal/astro_filter.py

```diff
@@ -79,7 +79,7 @@ def __init__(self,
         global scipy
 
         # check that we were passed a file and it exists
-        if type(filename) == type(''):
+        if type(filename) == type(str('')):
```

Comment:
This change doesn't quite make sense to me.  What is the value of filename that is causing you trouble?  Your proposed change is going to break an important input check.

Reply:
mmmm in my case if instances were not working because type('') returned "unicode". I just printed type(filename) across all the script and noticed it was always an string instance. So this modification worked for me

Reply:
This was my problem, even with filters format, directory, and call adress were supposedly correct
ValueError: Please pass the filename for the filter transmission file!